src/include/daos_srv/smd.h

```diff
@@ -197,9 +197,10 @@ int smd_pool_get_blob(uuid_t pool_id, int tgt_id, uint64_t *blob_id);
  * Get pool info, caller is responsible to free list items
  *
  * \param [OUT]	pool_list	Pool list
+ * \param [OUT] pool_cnt	Number of pools in list
```

Comment:
(style) There appears to be an alignment issue here.

Reply:
will fix this in this patch